src/pages/fr/core-concepts/astro-components.md

```diff
@@ -0,0 +1,388 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Composants
+description: Une introduction à la syntaxe des composants en .astro.
+---
+
+**Les composants Astro** sont les blocs de base de tout projet Astro. Il permettent de mettre en forme uniquement avec du HTML, sans code exécuté sur le navigateur.
+
+La syntaxe des composants Astro est une surcouche du HTML. Elle a été conçue pour [ressembler à HTML et JSX](/fr/comparing-astro-vs-other-tools/#astro-vs-jsx), et ajoute la possibilité d'inclure des composants et des expressions JavaScript. Vous pouvez identifier un composant Astro par son extension de fichier : `.astro`.
+
+Les composants Astro sont extrêmement flexibles. Il y a souvent des composants qui contiennent des **UI réutilisables sur la page**, comme un header ou un profil. D'autres composants peuvent contenir un morceau de HTML, comme un ensemble de balises `<meta>` qui facilitent la SEO. Les composants Astro peuvent aussi contenir une mise en page entière (appelée _Layout_).
+
+La chose la plus importante à savoir sur les composants Astro est qu'ils **produisent leur rendu HTML durant la compilation**. Cela signifie que si vous utilisez du JavaScript dans vos composants, ils seront tous exécutés avant la compilation. Le résultat sera un site plus rapide, avec aucun chargement de JavaScript ajoutée par défaut.
```

Comment:
```suggestion
La chose la plus importante à savoir sur les composants Astro est qu'ils **produisent leur rendu HTML durant la compilation**. Cela signifie que même si vous utilisez du JavaScript dans vos composants, tout sera exécuté pendant la compilation. Le résultat sera un site plus rapide, avec aucun chargement de JavaScript ajouté par défaut.
```
Revue de la traduction de "Even if you run JavaScript code inside of your components, it will all run ahead-of-time". Le pluriel ne fonctionnait pas vu qu'on parle du code (singulier).
En plus, pourquoi pas garder le "même" pour "even"